### PR TITLE
Scheduler profile & independent summary card component

### PR DESCRIPTION
**Pull request by YiranCdr**

### CODE REVIEW

File: app/actions/cpacpy.js [hunk 1]
```diff
@@ -168,13 +168,14 @@ export function scheduleDataSettings(scheduler, dataSettings, { success, error }
   )
 }
 
-export function schedulePipeline(scheduler, dataConfig, pipeline, { success, error }){
+export function schedulePipeline(scheduler, dataConfig, pipeline, { success, error }, profile= {}){
```
Comment (anibalsolon) [inline app/actions/cpacpy.js]: Just for conformity, I would suggest moving the callbacks to the last position, leaving 'profile' after 'pipeline'. Also, the profile should not be optional (so no `= {}`).

File: app/components/ExecutionNewPageSummary.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+  DatasetSiteIcon,
+  DatasetSessionIcon,
+  SchedulerIcon,
+  SchedulerParamIcon,
+} from '../components/icons'
+
+import cpac from '@internal/c-pac'
+import {fromJS} from "immutable";
+
+class ExecutionNewPageSummary extends Component {
```
Comment (anibalsolon) [inline app/components/ExecutionNewPageSummary.js]: Wouldnt it be `ExecutionSummary` ? It is used in other places, besides the `new execution` page.

File: app/containers/ExecutionNewPage.js [hunk 3]
```diff
@@ -55,13 +55,15 @@ import {
...
   SchedulerIcon,
+  SchedulerParamIcon,
   NextIcon,
   PreviousIcon,
 } from '../components/icons'
 
 import Modal from '../components/Modal'
 import Content from '../components/Content'
 import Box from '../components/Box'
+import { default as SummaryCard} from '../components/ExecutionNewPageSummary'
```
Comment (anibalsolon) [inline app/containers/ExecutionNewPage.js]: `import SummaryCard from '../components/ExecutionNewPageSummary'` works the same way!

the `default as` is mainly for exporting something with a different name.

**Review by anibalsolon — CHANGES_REQUESTED** [review]
All looks good, thank you! Just some minor changes, nothing that impacts the app flow/state.

File: app/components/ExecutionNewPageSummary.js (hunk 2 above)
Comment (YiranCdr) [reply app/components/ExecutionNewPageSummary.js]: The class name is now `ExecutionSummary`.

**Pull request merged**